app/services/imports/sales_logs_import_service.rb

```diff
@@ -16,7 +16,9 @@ def create_logs(folder)
   private
 
     def create_log(xml_doc)
+      # only import sales logs from 22/23 collection period onwards
       return unless meta_field_value(xml_doc, "form-name").include?("Sales")
+      return unless compose_date(xml_doc, "DAY", "MONTH", "YEAR") > Time.zone.local(2022, 4, 1)
```

Comment:
Is it possible to infer this date somehow?

Reply:
Hm, I thought about that, but not really. I think our form dates are based on `current collection period`, so even if I got the `previous collection start year` it would change in a month. This would always be the same tho